resources/assets/components/utilities/SiteFooter/SiteFooter.js

```diff
@@ -96,74 +110,189 @@ const SiteFooter = () => {
           <h4>Who We Are</h4>
           <ul>
             <li>
-              <a href="https://join.dosomething.org/">
+              <a
+                href="https://join.dosomething.org/"
+                onClick={event =>
+                  handleFooterTracking('whoWeAre', event.target.href)
```

Comment:
Our [analytics helper](https://github.com/DoSomething/phoenix-next/blob/03716ad136a3d00b5a31e36bf6bceba99e306118/resources/assets/helpers/analytics.js#L310) will snake case these event names, but for clarity perhaps worth inputting them as snake case here as well so it's more obvious what the final value will look like? (Here and below in the other links).

```suggestion
                  handleFooterTracking('who_we_are', event.target.href)
```